monad-logger-aeson/package.yaml

```diff
@@ -48,12 +48,12 @@ tests:
     build-tools:
     - hspec-discover
     dependencies:
-    - aeson >=1.5.6.0 && <2.1.3.0
-    - aeson-diff >=1.1.0.10 && <1.2.0.0
-    - base >=4.14.0.0 && <5
-    - bytestring >=0.10.12.0 && <0.12.0.0
+    - aeson >=1.5.2.0 && <2.1.3.0
+    - aeson-diff >=1.1.0.9 && <1.2.0.0
+    - base >=4.13.0.0 && <5
+    - bytestring >=0.10.10.1 && <0.12.0.0
     - directory >=1.3.6.0 && <1.4.0.0
-    - hspec >=2.7.10 && <2.10.0
+    - hspec >=2.7.6 && <2.10.0
```

Comment:
Let's change this lower bound from `2.7.6` to `2.7.9` so that it matches `stack-lts-16.31.yaml`.

Reply:
Ah good catch.